ui/widgets/nuxeo-user-tag.js

```diff
@@ -117,9 +117,26 @@ import './nuxeo-tooltip.js';
           type: Boolean,
           value: false,
         },
+
+        /**
+         * aria-label for accessibility
+         */
+        ariaLabel: {
+          type: String,
+          reflectToAttribute: true,
+          observer: '_ariaLabelChanged',
+        },
       };
     }
 
+    ready() {
+      super.ready();
+      const ariaLabel = this.getAttribute('aria-label');
+      if (!ariaLabel && this.user) {
+        this.setAttribute('aria-label', this._computeLabel(this.user));
+      }
```

Comment:
If we are already reflecting ariaLabel to attribute, can't we just do a
```suggestion
      if (this.user) {
        this.ariaLabel = this._name(this.user);
      }
```
?

Also, shouldn't we be doing this on a user observer? If the user changes, we should update the aria label accordingly, if none is defined by the user.